examples/e2e/package.json

```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "devDependencies": {
     "@pact-foundation/pact-node": "^4.6.0",
+    "cli-color": "^1.1.0",
```

Comment:
This is to make the console nice and red when there is a Pact failure (see https://github.com/pact-foundation/pact-js/pull/23/files#diff-ad2d27cf5c00ddf29bdafb5718a2a771R108). Might be overkill, thoughts?